paddle/fluid/operators/mkldnn/pool_mkldnn_op.cc

```diff
@@ -126,6 +126,9 @@ class PoolMKLDNNGradOpKernel : public paddle::framework::OpKernel<T> {
     UpdatePadding(&paddings, global_pooling, 0, padding_algorithm, data_dims,
                   strides, ksize);
 
+    platform::PoolingMKLDNNHandler<T>::ComputeAdaptivePoolParameters(
```

Comment:
Should `UpdatePadding(&paddings, global_pooling, 0, padding_algorithm, data_dims,strides, ksize);` be changed to 
`UpdatePadding(&paddings, global_pooling, ctx.Attr<bool>("adaptive"), padding_algorithm, data_dims,strides, ksize);`, or it does not matter ?

Reply:
@lidanqing-intel I think they should stay seprate as UpdatePadding is a common function used for CPU and oneDNN kernels (perhaps GPU as well) and computeAdaptivePoolParameters is just aplicable to oneDNN kernel.